packages/math/base-elements.lua

```diff
@@ -809,10 +809,6 @@ elements.text = pl.class({
       if self.text == "-" then
         self.text = "−"
       end
-      if symbolDefaults[self.text] then
-        self.atom = symbolDefaults[self.text].atom
-        self.stretchy = symbolDefaults[self.text].stretchy
-      end
```

Comment:
So that was the reason for the incorrect spacing of integrals… cheers to you, this was a 2-year-long standing bug :sweat_smile:

Reply:
Well thank you for putting me on the trail!